lib/dry/validation/extensions/struct.rb

```diff
@@ -0,0 +1,32 @@
+require 'dry-struct'
+
+module Dry
+  module Validation
+    class Schema
+      module StructClassBuilder
```

Comment:
@flash-gordon Why didn't you just created a class name `Struct < Dry::Struct` and at the test level just use that new class ?